### ISSUE TITLE
Model configurations in 150M models

### ISSUE THREAD

**Issue by freddy5566** [princeton-nlp/TRIME#5]
Hi,

I noticed that the configurations stated in the paper are different than the configuration in [wiki103-150M-trime.sh](https://github.com/princeton-nlp/TRIME/blob/main/train_scripts/wiki103-150M-trime.sh)

For example, the Learning rate in the paper is  2.5e-4; on the other hand, it is 0.0005 in the script. There are no Linear warmup steps in the paper; however, in the script, you set warmup steps to 8000. The Update steps and the batch size also seem to differ between the paper and the script. I followed the configuration in the script, yet, I couldn't reproduce the results of the transformer baseline (150M).

Could you guide me on how to correctly configure it to achieve a similar result?

Thank you.

**Comment by a3616001** [comment]
Hi, sorry for the confusion! The training recipe in the script should be the correct one. 
How many GPUs did you use when you trained the model? The effective batch size is 480 in this experiment.

**Comment by freddy5566** [comment]
hi, thanks for replying! I am using four GPUs, so I guess I can simply follow the training script. Another issue is whether you used the evaluation script for baseline models on both the dev and test set as follows,

```
python eval_lm-trime.py data-bin/wikitext-103 \
    --path pretrained_models/wiki103-247M-trime/checkpoint_best.pt \
    --sample-break-mode complete --max-tokens 3072 --context-window 2560 \
    --softmax-batch 1024 --gen-subset valid --fp16 \
    --max-sentences 1 --knn-keytype last_ffn_input \
    --use-local --softmax-temp 1.17
```

**Comment by a3616001** [comment]
Yes, we use the same script on both the dev set and the test set -- we tune the hyper-parameter (i.e., `--softmax-temp` in this case) on the dev set and use the same evaluation setup on the test set.
